spec/datadog/tracing/contrib/mongodb/client_spec.rb

```diff
@@ -154,6 +154,7 @@
         expect(span.get_tag('out.port')).to eq(port)
         expect(span.get_tag(Datadog::Tracing::Metadata::Ext::TAG_COMPONENT)).to eq('mongodb')
         expect(span.get_tag(Datadog::Tracing::Metadata::Ext::TAG_OPERATION)).to eq('command')
+        expect(span.get_tag(Datadog::Tracing::Metadata::Ext::TAG_KIND)).to eq('client')
```

Comment:
Prefer `span.kind` to avoid indirection in test 😄 

```suggestion
        expect(span.get_tag('span.kind')).to eq('client')
```